### PR TITLE
delete button bug fixed

### PR DESCRIPTION
**Pull request by Rus9688**

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -91,19 +91,35 @@ const listeners = {
...
+      if (document.querySelector("#chat-messages").classList.contains('selectMode') == false) {return false }
+     
+      const allCheckboxes = document.querySelectorAll('.checked')
+      
       if (document.querySelectorAll('.checked').length == 0) {
         document.getElementById("chatContent").dispatchEvent(
           new CustomEvent('chat:toggle-selection-mode', { detail: { chatType: e.detail.chatType } })
         )
       }
+      const deleteButton = document.getElementById("delete-btn");
```
Comment (salseeg) [inline assets/js/app.js:110]: this functionality should work for rooms too
are we sure that we have only one #delete-btn?

File: assets/js/app.js [hunk 2]
```diff
@@ -91,19 +91,35 @@ const listeners = {
...
+      const icon = document.querySelector('.x-icon');
+      const deleteSpan = document.getElementById('delete-span');
+      if (Array.from(allCheckboxes).some(el => el.previousElementSibling.classList.contains('justify-start')) ) {
```
Comment (salseeg) [inline assets/js/app.js:113]: use `x-mine` and `x-by-peer`
do not bind logic to layout classes

File: lib/chat_web/live/main_live/index.html.heex [hunk 3]
```diff
@@ -780,17 +780,17 @@
...
-                    <a class="cursor-pointer inline-flex items-center"
+                    <button id="delete-btn" class="cursor-pointer inline-flex items-center disabled:pointer-events-none"
                       phx-click={show_modal("delete-messages-popup") 
-                                 |> JS.set_attribute({"phx-click", hide_modal("delete-messages-popup") 
-                                                                   |> JS.push("room/delete-messages")
-                                                                   |> stringify_commands()},
-                                                                   to: "#delete-messages-popup .deleteMessageButton") 
-                                 |> JS.dispatch("chat:messages-to-delete", to: "#delete-messages-popup") }
+                                |> JS.set_attribute({"phx-click", hide_modal("delete-messages-popup") 
+                                                                  |> JS.push("dialog/delete-messages")
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/index.html.heex:786]: it should stay "room/delete-messages"

Comment (Rus9688) [reply lib/chat_web/live/main_live/index.html.heex]: fixed

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**